RELEASING.md

```diff
@@ -18,74 +18,58 @@ Also check if you can upgrade any dependencies:
 make update-dependency-versions
 ```
 
-## Make the release ##
+## Decide what the next version should be ##
+
+This depends on what's changed (see `CHANGELOG.md`):
 
-In order to make releases secrets are required. Members of the core team can
-install keybase and join the `cucumberbdd` team to access these secrets.
+* Bump `MAJOR` if:
+  * There are `Changed` or `Removed` entries
```

Comment:
This depends on the change. Not all changes have to be breaking changes.

Reply:
I copied the block from https://github.com/cucumber/cucumber/blob/master/RELEASE_PROCESS.md#decide-what-the-next-version-should-be

Reply:
Then that's wrong too. 😄

Reply:
Indeed 🙂
Once we agree on this pull request, I can make the same changes in other repositories.
I changed it to reference semver. Sounds better?